rbac/server/api/users.py

```diff
@@ -65,9 +66,22 @@
 
 @USERS_BP.post("api/users")
 async def create_new_user(request):
+    """ This API is invoked when a new User Account is Created"""
+    LOGGER.info("users.py: POST /api/users - A new user is trying to sign-up.")
     required_fields = ["name", "username", "password", "email"]
     utils.validate_fields(required_fields, request.json)
-
+    username_created = request.json.get("username")
+    conn1 = await db_utils.create_connection(
+        request.app.config.DB_HOST,
+        request.app.config.DB_PORT,
+        request.app.config.DB_NAME,
+    )
+    if await users_query.fetch_username_match_count(conn1, username_created) > 0:
+        LOGGER.info(
```

Comment:
Please remove logging statement.  This is a standard error that is directed to the client not an internal processing error